docs/guidance/kuberay-with-MCAD.md

```diff
@@ -14,83 +14,101 @@ MCAD allows you to deploy Ray cluster with a guarantee that sufficient resources
+      Type:                        HeadOfLine
+      Last Transition Micro Time:  2023-08-29T02:50:18.902379Z
+      Last Update Micro Time:      2023-08-29T02:50:18.902379Z
+      Reason:                      AppWrapperRunnable
+      Status:                      True
+      Type:                        Dispatched
+    Controllerfirsttimestamp:      2023-08-29T02:50:18.829462Z
+    Filterignore:                  true
+    Queuejobstate:                 Dispatched
+    Sender:                        before manageQueueJob - afterEtcdDispatching
+    State:                         Running
+  Events:                          <none>
+  (base) asmalvan@mcad-dev:~/mcad-kuberay$ kubectl get pod -n default
+  NAME                                           READY   STATUS    RESTARTS   AGE
+  raycluster-complete-head-9s4x5                 1/1     Running   0          47s
+  raycluster-complete-worker-small-group-4s6jv   1/1     Running   0          47s
+  ```
+
+- As seen the cluster is dispatched and pods are running. Let's submit another Ray cluster with AppWrapper CR and see it queued without creating pending pods using the command:
+  ```bash
+  kubectl create -f https://raw.githubusercontent.com/project-codeflare/multi-cluster-app-dispatcher/quota-management/doc/usage/examples/kuberay/config/aw-raycluster-1.yaml
+
+  # check the raycluster-complete-1 appwrapper
+  kubectl describe  appwrapper raycluster-complete-1 -n default
+  ``````
+  Expect
+  ```
+  Status:
+    Conditions:
+      Last Transition Micro Time:  2023-08-29T17:39:08.406401Z
+      Last Update Micro Time:      2023-08-29T17:39:08.406401Z
+      Status:                      True
+      Type:                        Init
+      Last Transition Micro Time:  2023-08-29T17:39:08.406452Z
+      Last Update Micro Time:      2023-08-29T17:39:08.406451Z
+      Reason:                      AwaitingHeadOfLine
+      Status:                      True
+      Type:                        Queueing
+      Last Transition Micro Time:  2023-08-29T17:39:08.423208Z
+      Last Update Micro Time:      2023-08-29T17:39:08.423208Z
+      Reason:                      FrontOfQueue.
+      Status:                      True
+      Type:                        HeadOfLine
+      Last Transition Micro Time:  2023-08-29T17:39:08.439753Z
+      Last Update Micro Time:      2023-08-29T17:39:08.439753Z
+      Message:                     Insufficient resources to dispatch AppWrapper.
+      Reason:                      AppWrapperNotRunnable.
+      Status:                      True
+      Type:                        Backoff
+    Controllerfirsttimestamp:      2023-08-29T17:39:08.406399Z
+    Filterignore:                  true
+    Queuejobstate:                 Backoff
+    Sender:                        before [backoff] - Rejoining
+    State:                         Pending
+  Events:                          <none>
+  ```
+
+  As seen the second Ray cluster is queued with no pending pods created.
```

Comment:
Every user allocates different CPU/memory resources to their Kubernetes clusters. If a user possesses a high-end workstation, would the RayCluster still be queued?

Reply:
No likely won't be queue. My cluster has only has 16 cpus with some overhead already allocated so the 2nd AW queued. If the high-end workstation has 16+ cpus, the 2nd AppWrapper may not go over total allocatable cpus.

Reply:
We should provide a clear example so users can consistently reproduce the expected behavior across all environments.

Reply:
Could you also explain when MCAD creates the RayCluster CR (e.g., 5 CPUs)? Without this information, it's difficult for users to understand how MCAD operates.

Reply:
@kevin85421 I have added more commits. Do you think these 2 comments have been addressed ?